src/main/resources/templates/rewards/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org/">
+<head th:replace="fragments :: head"></head>
+<body class="container">
+
+<header th:replace="fragments :: header"></header>
+
+<p th:unless="${rewards} and ${reward.size()}">No rewards!</p>
```

Comment:
`<p th:unless="${rewards} and ${rewards.size()}">No rewards!</p>`

Reply:
This has been updated on my local machine!